src/amber/controller/filters.cr

```diff
@@ -18,13 +19,14 @@ module Amber::Controller
         if self.responds_to? :after_filters
           self.after_filters
           @filters.run(:after, action)
+          @filters.run(:after, :except, action)
```

Comment:
Thanks for following the pattern here.

It seems like all of these `.run` statements could just live together in a single run method. Is that a worthy refactor @drujensen ?

Reply:
yeah, they could.  Up to @AndyRosenberg if he has time to do it.  If not, we can do it later.

Reply:
Don't think I'll be able to get to this one yet.